src/tla/Option.tla

```diff
@@ -0,0 +1,163 @@
+------------------- MODULE Option -----------------------
+(**
+ * Operators leveraging Apalache Variants to implement option types.
+ *
+ * Option types are a common algebraic data type used to represent values of a
+ * partial computation. See https://en.wikipedia.org/wiki/Option_type
+ *
+ * Igor Konnov, Informal Systems, 2021-2022
+ *)
+
+EXTENDS Apalache, Variants
+
+(* @typeAlias: option = Some(a) | None(UNIT); *)
+
+(**
+ * Lift a value of type `a` into an option.
+ *
+ * @param x a value
+ * @return the variant representing a value in the domain
+ *
+ * @type: a => $option; *)
```

Comment:
```suggestion
 * @type: a => $option;
 *)
```

I know that TLA+ does not have any *doc guidelines. But if we want to have anything that looks like Javadoc/Scaladoc, then `*)` should be aligned with the `*` above :)

Reply:
I formatted like to be consistent with how you formatted the docs in the `Variants.tla` module :D

Reply:
But nope. That was just my mistake. Will fix. :D